ckanext/portalopendatadk/templates/home/index.html

```diff
@@ -15,7 +15,7 @@
       {% snippet "home/layout{0}.html".format(homepage_style) %}
     {% endblock %}
   </div>
-<script>     
+{# <script>
     function setCookie(cname,cvalue,exdays) {
```

Comment:
@MuhammadIsmailShahzad I'm afraid this is another setCookie (and might have been used by something else (not the promo banner)), the promo banner js is here https://github.com/datopian/ckanext-portalopendatadk/blob/master/ckanext/portalopendatadk/fanstatic/promo.banner.js and it is not loaded, because the instruction to load it was in the  'snippets/promo-banners/influence-development.html', which you correctly commented out). So I'd vote for returning this setCookie function back unless you are absolutely sure it's not used anymore.

Reply:
@EvgeniiaVak Yes this is not the promo banner

This is the previous cookie which they were using before they started using the new one

Previously they were using
![Screenshot from 2020-10-06 19-21-28](https://user-images.githubusercontent.com/57398621/95336715-68d2e600-08ca-11eb-8c4a-d0861c73261c.png)

Now they are using the new one. Implemented in commit https://github.com/datopian/ckanext-portalopendatadk/commit/dbf87fa5083b2d103ed08ee3e33751e3832341c7
![Screenshot from 2020-10-06 17-35-20](https://user-images.githubusercontent.com/57398621/95336765-77210200-08ca-11eb-8c91-3db6d5dc6e82.png)

And they want the old one gone (The circle at bottom left is with the new cookie implemented)
![Screenshot from 2020-10-07 18-11-01](https://user-images.githubusercontent.com/57398621/95336799-81db9700-08ca-11eb-9c55-930a987fde9f.png)